### PR TITLE
Topic vineet updates

### PR DESCRIPTION
**Pull request by vineetgarc**
Assorted fixes - motivation being to add gcc git commit id to driver to show when doing arc64*gcc -v

### CODE REVIEW

**Review by claziss — APPROVED** [review]

File: Makefile.in [hunk 1]
```diff
@@ -56,7 +56,7 @@ ifneq ($(XLEN),32)
 GITVER := $(shell git --git-dir=$(GCC_SRCDIR)/.git describe --tag --always)
 PKGVER := "ARCv3 ARC64 GNU/Linux $(GITVER)"
 make_tuple = arc64-unknown-$(1)
-LINUX_TUPLE  ?= $(call make_tuple,linux-gnu)
+LINUX_TUPLE  ?= arc64-linux-gnu
```
Comment (cupertinomiranda) [inline Makefile.in:59]: I guess that historically we created "arc-snps-linux-" and then created "arc-linux-" soft links.
Should we change this ?

**Review by cupertinomiranda — APPROVED** [review]
All good for me, although I don't know if the tuple is proper or not.

File: Makefile.in (hunk 1 above)
Comment (claziss) [reply Makefile.in:59]: a tuple is something like `cpu-company-os-clibrary`. Maybe the best is just to remove the `unknown` from above.

**Review by shahab-vahedi — APPROVED** [review]

File: Makefile.in (hunk 1 above)
Comment (vineetgarc) [reply Makefile.in:59]: @cupertinomiranda indeed the ideal way is to do what you suggest, but the existing toolchain build scripts don't create any symlinks hence my fix to just remove "unknown". Note that more robust buildsystems (e.g. Buildroot) already do the right thing for arc64.

**Comment by vineetgarc** [comment]
Pushed !

**Pull request merged**